### ISSUE TITLE
Availability of a pretrained 3D model?

### ISSUE THREAD

**Issue by ajinkya-kulkarni** [funalab/QCANet#9]
Hi team, thanks for this repo! Do you have a pretrained 3D model (something like the StarDist2D model), which I can use out of the box to segment nuclei from my 3D dataset?

**Comment by sushmanthreddy** [comment]
@ajinkya-kulkarni  actually there is an issue with the code. I have been working on my dataset. there is an error from sckit image library. I was trying to contact to the team..but they arent responding...

@ajinkya-kulkarni  have u run this in ur local machine?

**Comment by funasoul** [comment]
Hi @ajinkya-kulkarni 
You can download the pretrained 3D model. It is described in the "Quickstart" section of README.md.
I have verified the command described in the "Quickstart" just works.

**Comment by sushmanthreddy** [comment]
@funasoul can u pls mention, how to train the model using c. Elegans data?. 
Does Pretrained model work c. Elegans data.. 

And pls can u mention version of frameworks used... 
There is some version issues , I was facing during using qcanet..

**Comment by tokkuman** [comment]
@sushmanthreddy The trained models (learned_nsn.npz, learned_ndn.npz) that are available are trained on mouse data.
The models trained on nematodes are ready to be available, so please wait a little longer.

The framework version is currently supported up to Chainer 7.7.0.
Please refer to the [requirements.txt](https://github.com/funalab/QCANet/blob/master/requirements.txt) file we have added for details.

**Comment by sushmanthreddy** [comment]
@tokkuman thanks for the reply.
And really thanks for updating the requirements.txt file, I have been trying to work with QCANet since last month. I have been using it for the c .elegans at the nuclei stage.
I have been trying to understand the [paper](https://www.nature.com/articles/s41540-020-00152-8#code-availability) clearly and, I am just confused about how to make a pre-trained model.
I am using the cell tracking challenge dataset for the training as mentioned in the QCANet paper.
These are my following doubts regarding the QCANet:
1. Is there any tool to make segmentation masks to create ground truth for the images? if it is there pls mention it here if possible.
2. Is pytorch branch created in the QCANet repo completely working can we use that? or we should use the master branch only?
3. Is there any particular way to contact you for my doubts an email? something would be really helpful to clear my doubts and contribute to QCANet.


Sir, I am new to open source, and I would love to contribute  QCANet for C .elegans data.

**Comment by tokkuman** [comment]
@sushmanthreddy 
1. You may want to use Fiji's Segmentation Editor plug-in, see https://imagej.net/plugins/segmentation-editor
2. We have confirmed that the minimum functionality works. However, we don't provide trained models, so you will need to create your own. See README for details.
3. The response frequency may not be that high, but if you need help please contact tokuoka@fun.bio.keio.ac.jp
